quiz167 (1).ipynb

```diff
@@ -0,0 +1,97 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "bb0ecdce-72d2-40a4-918a-88cd2a21e541",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "210c6d31-482f-4da3-b537-8c502b5d4978",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file = open(\"intro.txt\",'w',encoding='utf-8',newline='')"
```

Comment:
how to open this file to another way ?

Reply:
with open(\"intro.txt\",'w',encoding='utf-8',newline='')as file:

Reply:
WHAT IS "newline=''"? why we add it?

Reply:
Remove  for Extra Line And \n,\r